exporters/otlp/trace/build.gradle

```diff
@@ -3,27 +3,51 @@ plugins {
     id "maven-publish"
 
     id "me.champeau.gradle.jmh"
+    id "org.unbroken-dome.test-sets"
     id "ru.vyarus.animalsniffer"
 }
 
 description = 'OpenTelemetry Protocol Trace Exporter'
 ext.moduleName = "io.opentelemetry.exporter.otlp.trace"
 
+testSets {
+    testGrpcNetty
+    testGrpcNettyShaded
+    testGrpcOkhttp
+}
+
 dependencies {
     api project(':sdk:trace')
 
+    compileOnly "io.grpc:grpc-netty"
+    compileOnly "io.grpc:grpc-netty-shaded"
+
     implementation project(':sdk-extensions:otproto'),
             libraries.grpc_api,
             libraries.grpc_protobuf,
             libraries.grpc_stub,
             libraries.protobuf
 
     testImplementation project(':sdk:testing'),
-            'com.linecorp.armeria:armeria-grpc',
-            'com.linecorp.armeria:armeria-junit5',
             "io.grpc:grpc-testing:${grpcVersion}"
 
-    testRuntime "io.grpc:grpc-netty-shaded:${grpcVersion}"
```

Comment:
Since test sets automatically inherit the main test configuration, I needed to move this and the corresponding test out.